### DISCUSSION TITLE
How can I create a shortcode which just renders a shape?

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#11299] [Q&A] [answered]
I want to create a shortcode that accepts an alias, gets a content item by alias, and simply renders the shape. I can't seem to figure out how to do this. I've tried many variations of the following (note that for simplicity I have hard coded the alias):

`{% shape "InsightsCallout", alias: "test-callout" %}`

No matter what I do, however, I don't get anything being rendered to the page.

**Comment by Skrypt** [comment, marked as answer]
https://docs.orchardcore.net/en/latest/docs/reference/modules/Alias/#liquid

This should get you the ContentItem from the database with any alias you like.

**Reply by devlife** [reply]
Very interesting! Out of curiosity, why wouldn't the shape liquid template/filter work? I'll try out the link you sent and mark as answer when I get it to work.
